lib/graph3d/Graph3d.js

```diff
@@ -321,20 +326,23 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
 }
 
 
+/**
+ * Get the min/max values for the passed data column.
+ *
+ * Result is returned as a Range instance with min/max members
+ * properly set.
+ */
```

Comment:
I just noticed, that the parameters and return value is missing in this jsdoc header. So if you want to to be perfect you can add them: http://usejsdoc.org/tags-param.html